src/lib.rs

```diff
@@ -431,7 +441,7 @@ pub struct ElectrumOpts {
     pub timeout: Option<u8>,
     /// Sets the Electrum server to use
     #[structopt(
-        name = "SERVER:PORT",
+        name = "ELECTRUM_URL",
```

Comment:
5 lines below we have `pub electrum: String`. To keep it consistent with `esplora` maybe we can write `pub server: String`?

Reply:
Ah ya, not sure why I didn't change this one too, will fix.